app/serializers/score_serializer.rb

```diff
@@ -1,3 +1,3 @@
 class ScoreSerializer < ActiveModel::Serializer
-  attributes :id, :username, :text_id, :wpm, :time, :accuracy, :created_at, :updated_at
+  attributes :id, :username, :text_id, :wpm, :time, :accuracy, :position, :created_at, :updated_at
```

Comment:
@tiagonbotelho No, because `position` is a method 😛

It is inefficient in the way that for every score it does a query, but as we are limiting the scores query up to 5/10 scores I see no problem with that.

What do you think ?